docker-install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+sudo apt-get update
+sudo apt-get install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common
+
+sudo apt-key adv --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-keys 9dc858229fc7dd38854ae2d88d81803c0ebfcd88
+sudo add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
```

Comment:
Because I was used to install it in this way :)
Normally it is because ubuntu repo are updated very slowly and docker version installed from official apt repository is old, but with ubuntu 20, like you say, this is not the case, since on the default ubuntu 20.04 repo docker is at version `Docker version 19.03.8, build afacb8b7f0`, while we here install the latest (`19.03.12`).

Will fix this